src/V3Width.cpp

```diff
@@ -3498,14 +3507,15 @@ class WidthVisitor final : public VNVisitor {
                 if (classp->name() == "semaphore" || classp->name() == "process"
                     || VString::startsWith(classp->name(), "mailbox")) {
                     // Find the package the class is in
-                    AstNode* pkgItemp = classp;
-                    while (pkgItemp->backp() && pkgItemp->backp()->nextp() == pkgItemp) {
-                        pkgItemp = pkgItemp->backp();
-                    }
-                    AstPackage* const packagep = VN_CAST(pkgItemp->backp(), Package);
+                    AstPackage* const packagep = getItemPackage(classp);
                     // Check if it's std
                     if (packagep && packagep->name() == "std") {
-                        if (classp->name() == "semaphore" && nodep->name() == "get") {
+                        if (classp->name() == "process"
+                            && (nodep->name() == "self" || nodep->name() == "status"
+                                || nodep->name() == "kill" || nodep->name() == "await"
+                                || nodep->name() == "suspend" || nodep->name() == "resume")) {
+                            methodCallWarnTiming(nodep, "process");
```

Comment:
If we're warning on `self`, we may as well just warn on `process` and not worry about the methods. Then we won't have the monstrous OR.

Technically the random seed stuff doesn't require timing, but that's a limitation that could be relaxed in the future. (this could also be a comment here)

Reply:
Good idea, done.